020-Getting-started/070-nextjs.mdx

```diff
@@ -50,185 +46,621 @@ apiKey=YOUR_API_KEY_HERE
 
 ## Create a new Next.js app
 
-To start lets create a Next.js application. Set it up with typescript support so that we can utilize the SDK later in this tutorial.
+Let's begin by creating a new Next.js application:
```

Comment:
Can we explicitly say to accept all the defaults when asked? I feel it reduces a bit the choice anxiety.